### PR TITLE
Add component & styles

### PR DESCRIPTION
**Pull request by liannivin**

### CODE REVIEW

File: src/components/sendMessage/sendMessage.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import sendMessageCss from "./sendMessageCss";
+
+function SendMessage() {
+  return (
+    <form css={sendMessageCss} className="send-message">
+      <input
+        type="text"
+        id="js-input-user-message"
+        aria-label="Chat here"
+        className="input-text"
```
Comment (cdsandoval) [inline src/components/sendMessage/sendMessage.js]: You should delete this className if you are not using it.

File: src/components/sendMessage/sendMessage.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        placeholder="Chat here..."
+        autocomplete="off"
+        autofocus
+      />
+      <button id="js-add-user-message" className="button -dark submit">
```
Comment (cdsandoval) [inline src/components/sendMessage/sendMessage.js]: Check the id and className if you are not going to use it, must be delete.

File: src/components/sendMessage/sendMessage.js (hunk 1 above)
Comment (liannivin) [reply src/components/sendMessage/sendMessage.js]: Thanks for your recommendations

File: src/components/sendMessage/sendMessage.js (hunk 2 above)
Comment (liannivin) [reply src/components/sendMessage/sendMessage.js]: className are using Emotion so I did just delete `id`.

Comment (cdsandoval) [reply src/components/sendMessage/sendMessage.js]: Oh great. :+1:

File: public/assets/css/pages/main.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    display: flex;
+  }
+
+  .chat .list-messages {
+    left: 16px;
+    right: 16px;
+    bottom: 40px;
+    top: 16px;
+  }
+}
```
Comment (cdsandoval) [inline public/assets/css/pages/main.css]: You must delete this file.

**Pull request closed without merging**